src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -758,7 +758,8 @@ public boolean hasntStartedYet() {
         return isBuilding(); // there is no equivalent to a post-production state for flows
     }
 
-    synchronized @Nonnull List<SCMCheckout> checkouts(@CheckForNull TaskListener listener) {
+    @Exported
+    public synchronized @Nonnull List<SCMCheckout> checkouts(@CheckForNull TaskListener listener) {
```

Comment:
Since `SCMCheckout` is a package-private class, other plugins wouldn't be able to use this method without making that type public as well (You'd also need to add getters for its fields, etc.). I think it would be better to have a new method that returns `List<SCM>` (or maybe `List<? extends SCM>`, not sure what is normally used for this kind of thing) instead and populates that list using `WorkflowRun.checkouts` internally.